codalab/lib/server_util.py

```diff
@@ -226,7 +226,7 @@ def to_bytes(text, encoding='utf-8'):
     """Make sure text is bytes type."""
     if not text:
         return text
-    if not isinstance(text, bytes_type):
+    if not isinstance(text, bytes):
```

Comment:
I'm not sure if I'm missing something, but I don't see where `bytes` is defined. 

We also might need to add more tests around switching users and logging in and out to see if these changes are acceptable.

Reply:
`bytes` is a built-in in Python. I think `bytes_type` was only used because older version of oauthlib needed to keep Python 2 compatibility.

Reply:
I believe that only the CLI uses the oauth endpoints, and only for login and logout. I think they should be tested enough, given that we have a `cl logou` here?

https://github.com/codalab/codalab-worksheets/blob/35e65659ba48030b0cf17bf3fa0d71564e044253/test_cli.py#L265